### PR TITLE
Run tests on AOSP Evergreen platform

### PR DESCRIPTION
**Pull request by yuying-y**
b/268244534

Change-Id: Id54608490332514dc256e738ece58aa87855b94c

### CODE REVIEW

File: .github/config/evergreen-arm-softfp.json [hunk 1]
```diff
@@ -12,23 +24,26 @@
       "platform":"evergreen-arm-softfp",
       "target_platform":"evergreen-arm-softfp",
       "target_cpu":"target_cpu=\\\"arm\\\"",
-      "extra_gn_arguments":"use_asan=false"
+      "extra_gn_arguments":"use_asan=false",
+      "bootloader_extra_gn_arguments": "sb_is_evergreen_compatible=true",
```
Comment (andrewsavage1) [inline .github/config/evergreen-arm-softfp.json:28]: This should be set in the platform's configuration.gni, not passed through gn args

**Review by andrewsavage1 — COMMENTED** [review]
Could you rebase to kick off the builds again?

@isarkis would you take a quick look at the docker failure and see if it's an issue with the config needing to specify a different bootloader docker image vs just some kind of flake?

**Comment by isarkis** [comment]
> Could you rebase to kick off the builds again?
> 
> @isarkis would you take a quick look at the docker failure and see if it's an issue with the config needing to specify a different bootloader docker image vs just some kind of flake?

I think json config has syntax error, see https://screenshot.googleplex.com/3A2uq39Que6sZqR.png

File: .github/config/evergreen-arm-softfp.json (hunk 1 above)
Comment (andrewsavage1) [inline .github/config/evergreen-arm-softfp.json:28]: I think the comma at the end of this line is giving you issues, it's not valid JSON

**Comment by andrewsavage1** [comment]
@isarkis could you take a look at how I split up the evergreen config and comment?

**Review by isarkis — APPROVED** [review]
LGTM.  I wish we had a good way to define docker_service in includes, but it's not feasible ATM due to how main.yaml is written.  Maybe  in the future we should consider moving docker_service to includes if this becomes more than just a nuisance.

**Review by andrewsavage1 — APPROVED** [review]
This will require extra work with each of its cherry picks, as it corresponds to a past Starboard version instead of a Cobalt version, which is unusual for retroactive changes

**Pull request merged**